src/main/java/org/broadinstitute/hellbender/engine/AbstractConcordanceWalker.java

```diff
@@ -77,7 +77,7 @@ private void initializeTruthVariantsIfNecessary() {
         }
 
         if (truthVariants == null) {
-            truthVariants = new FeatureDataSource<>(new FeatureInput<>(truthVariantsFile, "truth"), CACHE_LOOKAHEAD, VariantContext.class);
+            truthVariants = new FeatureDataSource<>(new FeatureInput<>(truthVariantsFile, "truth"), CACHE_LOOKAHEAD, VariantContext.class, cloudPrefetchBuffer, cloudIndexPrefetchBuffer);
```

Comment:
@davidbenjamin You should make this same change to the `evalVariants` `FeatureDataSource`.

Reply:
Thanks @cmnbroad, did it.